core/templates.py

```diff
@@ -1,6 +1,8 @@
-from flask import request
+from flask import request, Markup
 from tcii18n.template import flask_methods
 
+import gfm
```

Comment:
I'm not sure but `gfm` import have to be with `flask` and `tcii18n` import or not ?